draft-ietf-quic-transport.md

```diff
@@ -2679,7 +2679,9 @@ endpoint receives.
 This design relies on the peer always sending a connection ID in its packets so
 that the endpoint can use the connection ID from a packet to reset the
 connection.  An endpoint that uses this design cannot allow its peers to send
-packets with a zero-length destination connection ID.
+packets with a zero-length destination connection ID and need to either use a
+the same connection ID length for all connections or ensure that the connection
+ID encodes its own length in a common fixed portion.
 
```

Comment:
It is not necessary to have a fixed portion. You can use bit 7 of each octet to indicate if there is more.

Reply:
Yeah, that might be a little more specific than is needed.  You could just say "...connection ID encodes its own length in a method the server can recover without state."

Reply:
Agreed, you could even use trial decryption if you really wanted :)